clip: geojson (#2243)
observablehq/plot: docs/features/marks.md

* **clip** - whether and how to clip the mark
* **tip** - whether to generate an implicit [pointer](../interactions/pointer.md) [tip](../marks/tip.md) <VersionBadge version="0.6.7" />

> If the **clip** option is *frame* (or equivalently true), the mark is clipped to the frame’s dimensions; if the **clip** option is null (or equivalently false), the mark is not clipped. If the **clip** option is *sphere* or a GeoJSON object with a polygonal geometry <VersionBadge pr="2243" />, then the mark will be clipped to the projected geometry (_e.g._, the front hemisphere when using the orthographic projection). A [geographic projection](./projections.md) is required for the sphere.

Reviewer: ```suggestion
If the **clip** option is *frame* (or equivalently true), the mark is clipped to the frame’s dimensions; if the **clip** option is null (or equivalently false), the mark is not clipped. If the **clip** option is *sphere*, then the mark will be clipped to the projected sphere (_e.g._, the front hemisphere when using the orthographic projection); a [geographic projection](./projections.md) is required in this case. Lastly if the **clip** option is a GeoJSON object <VersionBadge pr="2243" />, then the mark will be clipped to the projected geometry. 
```
